recipes/salib/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: salib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/SALib/SALib
+  git_tag: v{{ version }}
```

Comment:
Hi @jschueller - I am not comfortable merging in something that uses git sources, rather than a tarball.  Maybe you could figure out how to fix this or only clone the HEAD? Or maybe someone else is happy with it?

git does use a lot of CI resources that it isn't strictly necessary to use. Also, I have personally had to deal with this issue for other packages and so I suspect that a workaround can be found here.  Happy to take a look again if this becomes based on a tarball.  

Sorry, I know this can be sort of a hassle, but I do believe it will result in a better package in the long term.